### DISCUSSION TITLE
Installing version 4

### DISCUSSION THREAD

**Discussion by mr-easy** [agronholm/apscheduler#736] [Q&A]
Hi, 
Can anyone tell me the steps to build the version4, so that I can give it a try?
I am using apscheduler in my personal project, and I want to use the newer version.

**Comment by agronholm** [comment]
You don't need to build it, just install: `pip install --pre -U apscheduler`. If you want to install the latest version from git: `pip install git+https://github.com/agronholm/apscheduler`.
